lib/diagnostic.rb

```diff
@@ -136,25 +163,32 @@ def punch
     @moves << 'punch'
     @damage += 5
     @damage *= multiplier
+    self
   end
 
   def move(direction)
     @moves << "move #{direction}"
+    self
   end
 
   def kick
     @moves << 'kick'
     @damage += 10
     @damage *= multiplier
+    self
+  end
+
+  def self.get_possible_moves
+    'kick, move, punch'
   end
 
   private
 
   def multiplier
     case moves
-    when `%w`['punch', 'move left', 'kick']
+    when `%w{punch, move left, kick}` # ['punch', 'move left', 'kick']
```

Comment:
The grave accents will send this into the shell as a command.  Try inserting
```ruby
puts `ls`
```
into a Ruby file and running it.